README.md

```diff
@@ -135,3 +135,6 @@ By default, all providers in to the provider-database
 are assumed to support "Strict TLS".
 If that is not true for a specific provider,
 you have to add the optional top-level option `strict_tls: false`.
+
+Connections with `socket: PLAIN` are not affected by `strict_tls`,
```

Comment:
```suggestion
Connections with `socket: PLAIN` are not affected by the `strict_tls` option,
```
I'd suggest this change; without that, I didn't understand it without reading the context.